.github/workflows/release.yml

```diff
@@ -18,7 +18,10 @@ jobs:
           python-version: "3.x"
 
       - name: Build package
-        run: pipx run build
+        run: |
+          python -m pip install --upgrade pip
+          pip install build
+          python -m build
```

Comment:
What is the motivation to work outside a venv here?

Reply:
pipx is strictly for console scripts with an entry point, pip is more permissive and allows for libraries and other packages. I would not impose pipx for the packaging step.